app/src/main/kotlin/com/intive/picover/common/viewmodel/state/ViewModelState.kt

```diff
@@ -0,0 +1,19 @@
+package com.intive.picover.common.viewmodel.state
+
+sealed class ViewModelState<out T> {
```

Comment:
Was wondering about two things here:

- Adding undefined state
- If we should keep the data within the state -> i can imagine some more complicated view models with more values than one + it could be problematic for Loading Send state